packages/react-native-dogfood/Gemfile

```diff
@@ -8,4 +8,4 @@ gem "fastlane"
 plugins_path = File.join(File.dirname(__FILE__), 'fastlane', 'Pluginfile')
 eval_gemfile(plugins_path) if File.exist?(plugins_path)
 
-gem 'cocoapods', '~> 1.11', '>= 1.11.3'
+gem 'cocoapods', '~> 1.12', '>= 1.12.1'
```

Comment:
unrelated change but important, aligned with the one in podfile.lock to prevent future mishaps